language/move-lang/functional-tests/tests/libra/epilogue/charge_less_gas_for_less_work.move

```diff
@@ -5,31 +5,31 @@
 //! sender: alice
 //! gas-price: 1
 //! gas-currency: Coin1
-main() {
-    return;
+script {
+fun main() {
+}
 }
 // check: "Keep(EXECUTED)"
 
-
 //! new-transaction
 //! sender: bob
 //! gas-price: 1
 //! gas-currency: Coin1
-main() {
-    let x: u64;
-    x = 1;
-    while (copy(x) < 2000) { x = move(x) + 1; };
-    return;
+script {
+fun main() {
+    let x = 1;
+    while (x < 2000) { x = x + 1; };
```

Comment:
I don't think you need the braces here
`while (x < 2000)  x = x + 1`
Should work